### ISSUE TITLE
az iot hub device-twin show : --login argument requires account to be setup

### ISSUE THREAD

**Issue by photomoose** [Azure/azure-iot-cli-extension#243]
## Describe the bug

**Command Name**
`az iot hub device-twin show
Extension Name: azure-iot. Version: 0.9.8.`

**Errors:**
```
Please run 'az login' to setup account.
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- `az iot hub device-twin show --device-id <device id> --login <IOT Hub connection string>`

## Expected Behavior
The Device Twin should be returned without having to configure the CLI with an account.

## Environment Summary
```
Linux-4.19.76-linuxkit-x86_64-with-debian-10.5
Python 3.6.10
Installer: DEB

azure-cli 2.11.0

Extensions:
azure-iot 0.9.8

```

**Comment by digimaun** [comment]
Thanks for bringing this up. The issue has been fixed - it will be in the upcoming `0.9.9` release.

**Comment by digimaun** [comment]
We released [0.9.9](https://github.com/Azure/azure-iot-cli-extension/releases/tag/v0.9.9) on Aug 28th. Please give it a try. If you still run into the problem, you can comment here and we can re-open.
